requirements.txt

```diff
@@ -7,4 +7,7 @@ requests==2.5.1
 six==1.9.0
 utm==0.3.1
 wsgiref==0.1.2
-google-api-python-client==1.1
\ No newline at end of file
+google-api-python-client==1.1
+jsonpickle
```

Comment:
nit: This change does not require jsonpickle or python-crontab.
Also, don't you need to add urllib2?